user/getting-started.md

```diff
@@ -107,6 +110,11 @@ time="2017-07-12T14:11:13Z" level=debug msg="starting server"
 time="2017-07-12T14:11:13Z" level=debug msg="registering gRPC service"
 time="2017-07-12T14:11:13Z" level=info msg="starting gRPC server"
 ```
+
+Notice how in this case we need to share the host Docker server's Unix socket with
```

Comment:
I think there's a misconception here, or at least it's not clear enough to me with current explanation. With this example, it seems that you always need to share your docker daemon socket, and that you need to use `docker-daemon` transport to get your images.

This is neither necessary not the best option. The simplest way to use your custom images is to just let the server fetch the custom images from dockerhub instead:
```
$ BBLFSH_DRIVER_IMAGES="python=docker://bblfsh/python-driver:dev-136e550" docker run -e BBLFSH_DRIVER_IMAGES --privileged -p 9432:9432 --name bblfsh bblfsh/server
```

Using docker-daemon is just an additional, more complex way to get the same result that only makes sense if you already have the image handled by your docker (you have built it or similar). But this is a more advanced approach.